src/gsy_e/models/strategy/external_strategies/smart_meter.py

```diff
@@ -0,0 +1,438 @@
+# pylint: disable=broad-except
```

Comment:
We try not to disable pylint warnings on module basis. If you see occurrences, these are from the time before we discussed that this is not a good idea, please remove then, thanks!